src/Ocelot/Multiplexer/MultiplexingMiddleware.cs

```diff
@@ -215,7 +215,8 @@ protected virtual async Task<HttpContext> CreateThreadContextAsync(HttpContext s
         {
             Request =
             {
-                Body = await CloneRequestBodyAsync(source.Request, source.RequestAborted),
+                Body = await CloneRequestBodyAsync(source.Request, source.RequestAborted),
+                /* Form = from.Form, // TODO */
```

Comment:
No, Form is not required to be mapped here.

Form is just an interpretation of the body and copying the body is enough to make it work.

Acceptance test `Should_return_response_200_with_copied_form_sent_on_multiple_service` validates this point by sending a form, reading it and responding with data from it.

Reply:
I'm confused and surprised, but it's all good. If the Form property only reads the Body, there isn't much to discuss, except perhaps for Files. However, since Files is a sub-property of Form, I anticipate there will be no issues, and the files should be forwarded as well, correct?

Reply:
I hesitated to put a file in an acceptance test, but yeah everything relies on the body and how it should be interpreted through the application content type headers. So yes, correct as far as I know.

It was a nice surprise for me too :)

Reply:
If you are inspired by the file forwarding feature, feel free to update the relevant acceptance test or write a new one.
Thank you!